pkg/operator/storageclasscontroller/vmware_test.go

```diff
@@ -174,7 +173,7 @@ func TestZonalPolicyCreation(t *testing.T) {
 				attach_tag_api:      1,
 				create_profile_api:  1,
 			},
-			featureGates: featuregates.NewFeatureGate([]v1.FeatureGateName{"SomeEnabledFeatureGate"}, []v1.FeatureGateName{"SomeDisabledFeatureGate", features.FeatureGateVSphereMultiVCenters}),
+			featureGates: featuregates.NewFeatureGate([]v1.FeatureGateName{"SomeEnabledFeatureGate"}, []v1.FeatureGateName{"SomeDisabledFeatureGate"}),
 		},
```

Comment:
If we are not using `featureGates` field, why not remove it from the test struct?